benchmarks/domains/direct_arylation/direct_arylation_multi_batch.py

```diff
@@ -26,83 +27,85 @@ def direct_arylation_multi_batch(
     settings: ConvergenceBenchmarkSettings,
 ) -> pd.DataFrame:
     """Benchmark function for direct arylation reaction."""
-    data = pd.read_table(
-        DATA_PATH / "direct_arylation" / "data.csv",
-        sep=",",
-        index_col=0,
-    )
-    non_substance_paramters = [
-        NumericalDiscreteParameter(
-            name="Concentration",
-            values=sorted(data["Concentration"].unique()),
-        ),
-        NumericalDiscreteParameter(
-            name="Temp_C",
-            values=sorted(data["Temp_C"].unique()),
-        ),
-    ]
-    catgorical_parameters = [
-        CategoricalParameter(name=substance, values=data[substance].unique())
-        for substance in ["Solvent", "Base", "Ligand"]
-    ]
-    rdkit_parameters = [
-        SubstanceParameter(
-            name=substance,
-            data=dict(zip(data[substance], data[f"{substance}_SMILES"])),
-            encoding="RDKIT2DDESCRIPTORS",
+    with temporary_seed(settings.random_seed):
```

Comment:
TODO: Instead of wrapping whole function in random seed, set random seed specifically for where data is generated at random